sqlserver/datadog_checks/sqlserver/data/conf.yaml.example

```diff
@@ -235,6 +235,23 @@ instances:
         #
         # collection_interval: 10
 
+    ## Configure how the SQL obfuscator behaves.
+    #
+    obfuscator_options:
+
+        ## @param replace_digits - boolean - optional - default: false
+        ## Set to `true` to replace digits in identifiers and table names with question marks in your SQL statements.
+        ## Note that this option only applies when `dbm` is enabled.
+        #
+        # replace_digits: false
+
+        ## @param collect_comments - boolean - optional - default: true
+        ## Set to `false` to disable the collection of SQL comments. Disabling this option can interfere with
+        ## database monitoring features working correctly.
+        ## Note that this option only applies when `dbm` is enabled.
```

Comment:
should this doc mention _what_ features they get if they enable this?

Reply:
I personally think we should leave the specifics to our online docs and not in the config. It is a bit inconvenient to update every integration's  config doc anytime we add/remove something and having to wait until the next release cycle. It could also be misleading if a customer doesn't upgrade and they see a listed feature that was actually removed in the newer agents.

Reply:
gotcha - I just think mentioning the feature wouldnt work without saying what it is, is kind of confusing

Reply:
Yeah, I can agree with that. Tough, I'm not sure what the best approach is product wise. Let's move this discussion offline.

Reply:
https://github.com/DataDog/integrations-core/pull/10776/commits/998b89ad4da6952a765e3f44b9cada4dc942cf53